.gitattributes

```diff
@@ -1,5 +1,7 @@
 * text=auto
 
+*.bat text eol=crlf
+*.patch -text
```

Comment:
So the patches should already be untouched as `text=auto` should let it default to whatever you added. What evidence do you have that this is not the case?